proto/api/api.proto

```diff
@@ -223,9 +226,21 @@ message SuggestGasPriceResponse {
   uint64 gasPrice = 1;
 }
 
+// To be deprecated
 message EstimateGasForActionRequest {
-  iotextypes.Execution execution = 1;
-  string callerAddress = 2;
+  iotextypes.Action action = 1;
+}
+
+message EstimateActionGasConsumptionRequest {
+  oneof action {
+    iotextypes.Transfer transfer = 1;
+    iotextypes.Execution execution = 2;
+  }
+  string callerAddress = 3;
```

Comment:
string callerAddress = 100; in case we will add more action type to oneof, which could have continuous number

Reply:
sure